app.rb

```diff
@@ -15,14 +15,14 @@ def create_person_inputs
     print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
     type_num = gets.chomp.to_i
     if type_num == 1
-        type = 'Student'
+      type = 'Student'
     elsif type_num == 2
-        type = 'Teacher'
+      type = 'Teacher'
     else
-        puts 'Invalid option'
-        return
+      puts 'Invalid option'
+      return
     end
-    print "Age: "
+    print 'Age: '
     age = gets.chomp.to_i
     print 'Name: '
     name = gets.chomp.capitalize
```

Comment:
![image](https://github.com/AliRezaBmeDu/school-library/assets/93039359/a58decae-5ed0-4263-83eb-4be4dad87c6c)

- [ ] It would be nice if your app asked for parent_permission while creating a student :+1: because right now it is not asking it and before making the changes your app was taking input for parent_permission :thinking: Kindly fix this problem omake your app work as expected :blush: